ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -732,8 +785,28 @@ public class BottomSheetController: UIViewController, Shadowable, TokenizedContr
             sheetHeight = expandedSheetHeight
         }
 
-        return CGRect(origin: CGPoint(x: (view.bounds.width - sheetWidth) / 2, y: offset),
-                      size: CGSize(width: sheetWidth, height: sheetHeight))
+        let padding: CGFloat = 8
+
+        let xPosition: CGFloat
+        if anchoredEdge == .trailing {
+            if UIView.userInterfaceLayoutDirection(for: view.semanticContentAttribute) == .leftToRight {
+                xPosition = view.bounds.width - sheetWidth - padding
+            } else {
+                xPosition = padding
+            }
+        } else if anchoredEdge == .leading {
+            if UIView.userInterfaceLayoutDirection(for: view.semanticContentAttribute) == .leftToRight {
+                xPosition = padding
+            } else {
+                xPosition = view.bounds.width - sheetWidth - padding
+            }
+        } else {
+            xPosition = (view.bounds.width - sheetWidth) / 2
+        }
```

Comment:
Not sure if there is a style preference for Fluent, but you may consider using a switch statement here since all of the `BottomSheetAnchorEdge` values need to be covered. Also adding a short comment describing how the `xPosition` is determined may be helpful.